frontend/src/metabase/ui/components/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -1,10 +1,16 @@
 import type { MultiSelectProps, SelectItem } from "@mantine/core";
-import { MultiSelect } from "@mantine/core";
+import { MultiSelect, Tooltip } from "@mantine/core";
```

Comment:
I found some edge cases. I think we should fix at least Case 3 since it results in React going crazy.

### Case 1

Type this: `Kamil "The QA" Mielnik,`
I'd expect that `,` triggers `onChange`.
Blur will trigger `onChange` but the value will include the comma (which isn't expected)


### Case 2

_similar to Case 1_

Paste this: `Kamil "The QA" Mielnik,`
`onChange` won't be triggered like I'd expect it to.

### Case 3

Type: `a,a`
Blur

There will be 2 `a` values but there should be no duplicates.
You can keep typing `a` and blurring and there will be more duplicates.
Things start to go crazy when there are duplicates because of non-unique `key` props (if you try to delete items with "x" button, etc.)

Reply:
I've fixed case 3 and found a way to relax the breakage around case 1 and 2.

Entering `Kamil "The QA" Mielnik` now Just Works™, but this introduced another edge case. Enter `foo "bar, baz" quu,`. Instead of one value `foo "bar, baz" quu` this creates two `foo "bar` and `baz" quu`. 

This is hard to fix without breaking other (more common) cases and can be worked around by the user typing the (more well-formed) string `"foo \"bar, baz\" quu"`.

@mngr what do you think, is this acceptable?